src/Abp.Web.Api/WebApi/Controllers/Dynamic/Selectors/HttpVerbExtensions.cs

```diff
@@ -80,6 +82,11 @@ public static HttpVerb ToHttpVerb(this HttpMethod method)
                 return HttpVerb.Head;
             }
 
+            if (method == new HttpMethod("PATCH"))
```

Comment:
That's good. I hadn't implement Patch since I could not find it (and sorry, I haven't searched for it much).
But I think it's better to directly check Method property, which is a string and proper for switch. I'll implement it. Thanks.